pom.xml

```diff
@@ -194,11 +194,71 @@
+                            <version>2.8.3</version>
+                        </exclude>
+                        <exclude>
+                            <groupId>org.springframework</groupId>
+                            <artifactId>spring-beans</artifactId>
+                            <version>5.3.12</version>
+                        </exclude>
+                        <exclude>
+                            <groupId>org.springframework</groupId>
+                            <artifactId>spring-web</artifactId>
+                            <version>5.3.12</version>
+                        </exclude>
+                        <exclude>
+                            <groupId>org.yaml</groupId>
+                            <artifactId>snakeyaml</artifactId>
+                            <version>1.28</version>
+                        </exclude>
+                        <exclude>
+                            <groupId>org.springframework</groupId>
+                            <artifactId>spring-webmvc</artifactId>
+                            <version>5.3.12</version>
+                        </exclude>
+                        <exclude>
+                            <groupId>ch.qos.logback</groupId>
+                            <artifactId>logback-classic</artifactId>
+                            <version>1.2.6</version>
+                        </exclude>
+                        <exclude>
+                            <groupId>org.apache.tomcat.embed</groupId>
+                            <artifactId>tomcat-embed-core</artifactId>
+                            <version>9.0.54</version>
+                        </exclude>
+                        <exclude>
+                            <groupId>org.springframework.boot</groupId>
+                            <artifactId>spring-boot-autoconfigure</artifactId>
+                            <version>2.5.6</version>
+                        </exclude>
+                        <exclude>
+                            <groupId>ch.qos.logback</groupId>
+                            <artifactId>logback-core</artifactId>
+                            <version>1.2.6</version>
+                        </exclude>
+                        <exclude>
+                            <groupId>com.google.code.gson</groupId>
+                            <artifactId>gson</artifactId>
+                            <version>2.8.8</version>
+                        </exclude>
+                        <exclude>
+                            <groupId>net.minidev</groupId>
+                            <artifactId>json-smart</artifactId>
+                            <version>2.4.7</version>
+                        </exclude>
+                        <exclude>
+                            <groupId>com.fasterxml.jackson.core</groupId>
+                            <artifactId>jackson-databind</artifactId>
+                            <version>2.12.5</version>
+                        </exclude>
```

Comment:
That's a lot of excludes.

Reply:
We should have pointers to tickets to either monitor or fix these. Particularly as there's an out of date comment above the first one.

Reply:
This was discussed at standup this morning. [A ticket has been created](https://jira.ons.gov.uk/browse/DIS-1209) to upgrade the Java version of this service, allowing upgrades to major versions of some of the vulnerable dependencies. This will reduce the number of vulnerabilities on the service.
A link to the ticket has been added in the pom.xml file